client/dashboard/me/billing-purchases/purchase-settings/index.tsx

```diff
@@ -0,0 +1,1209 @@
+		purchase.site_slug &&
+		purchase.meta
+	) {
+		return (
+			<Text>
+				{ createInterpolateElement(
+					__( 'You need to <a>start the domain transfer</a> for your domain.' ),
+					{
+						a: (
+							<a
+								href={ domainUseMyDomain(
+									purchase.site_slug,
+									purchase.meta,
+									useMyDomainInputMode.startPendingTransfer
+								) }
+							/>
+						),
+					}
+				) }
+			</Text>
+		);
+	}
+
+	if ( domain.transfer_status === DomainTransferStatus.CANCELLED ) {
+		return (
+			<Text>
+				{ createInterpolateElement( __( 'Transfer failed. Learn the possible <ReasonsWhy />.' ), {
+					LearnMore: (
+						<ExternalLink
+							href={ INCOMING_DOMAIN_TRANSFER_STATUSES_IN_PROGRESS }
+							rel="noopener noreferrer"
+						>
+							{ __( 'reasons why' ) }
+						</ExternalLink>
+					),
+				} ) }
+			</Text>
+		);
+	}
+
+	if ( domain.transfer_status === DomainTransferStatus.PENDING_REGISTRY && transferEndDate ) {
+		return (
+			<Text>
+				{ createInterpolateElement(
+					__(
+						'The transfer should complete by <TransferFinishDate />. We are waiting for authorization from your current domain provider to proceed. <LearnMore />'
+					),
+					{
+						TransferFinishDate: (
+							<strong>
+								{ formatDate( new Date( transferEndDate ), locale, { dateStyle: 'long' } ) }
+							</strong>
+						),
+						LearnMore: (
+							<ExternalLink
+								href={ INCOMING_DOMAIN_TRANSFER_STATUSES_IN_PROGRESS }
+								rel="noopener noreferrer"
+							>
+								{ __( 'Learn more' ) }
+							</ExternalLink>
+						),
+					}
+				) }
+			</Text>
+		);
+	}
+
+	if ( domain.transfer_status === DomainTransferStatus.PENDING_REGISTRY ) {
+		return (
+			<Text>
+				{ createInterpolateElement(
+					__(
+						'We are waiting for authorization from your current domain provider to proceed. <LearnMore />'
+					),
+					{
+						LearnMore: (
+							<ExternalLink
+								href={ INCOMING_DOMAIN_TRANSFER_STATUSES_IN_PROGRESS }
+								rel="noopener noreferrer"
+							>
+								{ __( 'Learn more' ) }
+							</ExternalLink>
+						),
+					}
+				) }
+			</Text>
+		);
+	}
+
+	if ( transferEndDate ) {
+		return (
+			<Text>
+				{ createInterpolateElement(
+					__( 'The transfer should complete by <TransferFinishDate />. <LearnMore />' ),
+					{
+						TransferFinishDate: (
+							<strong>
+								{ formatDate( new Date( transferEndDate ), locale, { dateStyle: 'long' } ) }
+							</strong>
+						),
+						LearnMore: (
+							<ExternalLink
+								href={ INCOMING_DOMAIN_TRANSFER_STATUSES_IN_PROGRESS }
+								rel="noopener noreferrer"
+							>
+								{ __( 'Learn more' ) }
+							</ExternalLink>
+						),
+					}
+				) }
+			</Text>
+		);
+	}
+
+	return null;
+}
+
+function PurchaseSecondSubtitle( { purchase }: { purchase: Purchase } ) {
+	if ( purchase.is_domain ) {
+		if ( purchase.bill_period_days === SubscriptionBillPeriod.PLAN_CENTENNIAL_PERIOD ) {
+			return (
+				<Text variant="muted">
+					{ ' ' }
```

Comment:
Is this expected?

Reply:
No, I think that was a typo. Thanks for noticing!